platforms/software/version-control/src/main/java/org/gradle/vcs/git/GitVersionControlSpec.java

```diff
@@ -25,6 +26,7 @@
  *
  * @since 4.4
  */
+@NotToBeMigratedToLazy
 public interface GitVersionControlSpec extends VersionControlSpec {
```

Comment:
❓ Shouldn't we migrate this?

Reply:
Ah, so we decided not to migrate the whole of `:version-control`? It seems there's only a few properties here, should we reconsider?

Reply:
Yeah, not sure. I can add annotation for migration. I am not entirely sure what we will do with that